server/protocols/jmap-draft-integration-testing/memory-jmap-draft-integration-testing/src/test/java/org/apache/james/jmap/memory/MemorySendMDNMethodTest.java

```diff
@@ -39,6 +40,7 @@
 
     @RegisterExtension
     JamesServerExtension testExtension = new JamesServerBuilder()
+        .extension(new MemoryCommonExtension())
```

Comment:
Regarding just custom guice extensions, don't it make sens to limit the count of retries only when doing integration testing for dead-letter?

This would make the changeset way less verbose and I will not have to configure a test-jar import on each and every James server I launch...

Reply:
> don't it make sens to limit the count of retries only when doing integration testing for dead-letter

Then you still suffer 5h each build, some reach to 6h

Reply:
then the problem is SpamAssassin, don't you think?

Reply:
Very true, thanks! I had an idea that is creating a new NOOP SpamAsssassin and use it in the integration tests. James doesn't have to retry at all.